tests/dialects/test_stablehlo.py

```diff
@@ -0,0 +1,31 @@
+from typing import TYPE_CHECKING, reveal_type
```

Comment:
is reveal_type only available when TYPE_CHECKING?

Reply:
Mmm... I'm not super familiar with pyright nor reveal_type. 😅 But reveal_type didn't like the expected_text keyword argument when executing at runtime, but it is totally fine with it at pyright time.